### PR TITLE
Update All files

### PR DESCRIPTION
**Pull request by josuerushanika**
In this milestone of Raect to-do list we have added the following requirements :

add a task to the header
edit a task
remove a task
check a task as done
use react rooters
Create the menu of the app and the hamburger dropdown
Style to-do list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# React-with-toda
```
Comment (mertkantaroglu) [inline README.md:1]: Please don't forget to add the README file, so if someone visits your project they can see the details about your project and yourself.

File: src/reportWebVitals.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (mertkantaroglu) [inline src/reportWebVitals.js:15]: In this project, this file is not necessary. Therefore, you can delete this file and erase the related codes from index.js in order to keep your project cleaner.

File: src/Components/TodoItem.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+          onChange={() => {
+            handleChange(itemProp.id);
+          }}
+        />
+        <button
+          type="button"
+          onClick={handleEditing}
+        >
+          Edit
+        </button>
```
Comment (mertkantaroglu) [inline src/Components/TodoItem.js:58]: Please consider fixing the position of this Edit button. 

Screen Shot 2023-03-08 at 16 03 43

**Review by mertkantaroglu — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @josuerushanika ,

You have done good job so far! 👍 

There are some issues that you still need to work on to prepare your project for the approval but you are almost there! 

## Top Highlights
✅ Descriptive PR 👍
✅ Nice design 🔥
✅ Linters are passing 👏
✅ Correct functionality 🙌

almost_there_dog

## Optional suggestions

- [ ] You may consider to switch the Button names with button icons. Your project would look much nicer.
- [ ] Naming of the project seems not exactly correct. 'Todo with React' would make more sense.

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/reportWebVitals.js (hunk 2 above)
Comment (josuerushanika) [reply src/reportWebVitals.js:15]: @mertkantaroglu  I think  this file is necessary to run my project

File: src/Components/TodoItem.js (hunk 3 above)
Comment (josuerushanika) [reply src/Components/TodoItem.js:58]: @mertkantaroglu   The button work perfectly

**Review by mertkantaroglu — APPROVED** [review]
Status: APPROVED 🟢 
Hi, @josuerushanika ,

Good job so far 👍 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

minions

**Pull request merged**